File: src/test/java/de/fraunhofer/aisec/codyze/crymlin/IssueTest.java

package de.fraunhofer.aisec.codyze.crymlin;

import org.junit.jupiter.api.Test;

class IssueTest extends AbstractMarkTest {

	@Test
	void issue219() throws Exception {
		var findings = performTest("issues/219/Main.java", "issues/219/");
		findings.forEach(System.out::println);
	}

}

File: src/codyze_legacy/src/test/java/de/fraunhofer/aisec/codyze/crymlin/IssueTest.kt
package de.fraunhofer.aisec.codyze.crymlin

import java.lang.Exception
import kotlin.Throws
import org.junit.jupiter.api.Test

class IssueTest : AbstractMarkTest() {

    @Test
    @Throws(Exception::class)
    fun issue219() {
        val findings = performTest("legacy/issues/219/Main.java", "legacy/issues/219/")

        expected(findings, "line 6: Rule JCAProvider_PBEParameterSpec_2 violated")
    }
}
